contracts/.openzeppelin/project.json

```diff
@@ -9,16 +9,18 @@
   "version": "0.0.1",
   "compiler": {
     "compilerSettings": {
+      "evmVersion": "berlin",
```

Comment:
Where was this set to `berlin`? I don't see that setting defined.

Also, why `berlin`? The [Solidity docs](https://solidity.readthedocs.io/en/v0.6.12/using-the-compiler.html#target-options) say it's experimental so I'm not sure if that's a good idea. Is there any reason not to just use the default of `petersburg`?

Reply:
OK good call on this. I had bumped it up because GSN using `chainid` which is only available in `istanbul` or greater, but I was wrong to jump straight to Berlin. Fixed this.